docs/InterfaceKit.md

```diff
@@ -1,3 +1,135 @@
+|[phidget.quit](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md#methods)|N/A |This method requests a disconnect from the phidget board.  The disconnected event will be dispatched when the connection has been successfully disconnected.|
+|whenReady|function|This executes a function when the Phidget InterfaceKit is ready to be used.|
+|observeOutputs|outputs|Used for asynchronously observing the digital output changes to the Phidget InterfaceKit board. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
+|observeInputs|inputs|Used for asynchronously observing the digital input changes to the Phidget InterfaceKit board. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
+|observeSensors|sensors|Used for asynchronously observing the analog input changes with an A to D digitalization of 2<sup>10</sup> to the Phidget InterfaceKit board. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
+|observeRawSensors|rawSensors|Used for asynchronously observing the analog input changes with an A to D digitalization of 2<sup>12</sup> to the Phidget InterfaceKit board. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
+
+##Board
+
+Many different kinds of [Interface Boards](http://www.phidgets.com/products.php?category=0) are available. __board__ will give you info about the features of your board.
+
+|Array Name| Description|
+|---|---|
+|Output|Outputs available|
+|Input|Digital inputs available|
+|Sensor|Analog inputs available, 2<sup>10</sup> digitalization|
+|rawSensor|Analog inputs available, 2<sup>12</sup>digitalization|
+|Trigger|Input triggers available|
+|DataRate|Data rates in milliseconds|
+
+
+
+##Data
+
+|Key|Data Type|Writable|Description|
+|---|---|---|---|
+|type|string|no|'PhidgetInterfaceKit'|
+|dataRateMax|number|no|Maximum data rate in samples per second|
+|dataRateMin|number|no|Minimum data rate in samples per second|
+|dataRate|number|yes|The number of times data is acquired per second|
+|ratiometric|bool|yes| __True__ = configures the Analog Inputs to measure relative to VCC (nominal 5V), __False__ = configures the Analog Inputs to measure relative to an internal precision 5V reference|
```

Comment:
I believe VCC is the voltage you supply to the board correct? If this is the case it would be best to put that into the description. Most people wont know what it is nor will they understand what nominal voltage is.

True = configures the Analog Inputs to measure relative to the voltage supplied to the board, VCC.